database/aoai-proxy.sql

```diff
@@ -323,6 +353,21 @@ CREATE TABLE aoai.event_catalog_map (
 
 ALTER TABLE aoai.event_catalog_map OWNER TO admin;
 
+--
+-- Name: metric; Type: TABLE; Schema: aoai; Owner: admin
+--
+
+CREATE TABLE aoai.metric (
+    event_id character varying(50) NOT NULL,
+    metric_timestamp date NOT NULL,
+    api_key uuid NOT NULL,
```

Comment:
Why are we storing `api_key`? Doesn't that represent a security risk? Wouldn't `user_id` be better?

Reply:
`api_key` is a unique value within the `event_attendee` table so the right one to use.